### ISSUE TITLE
Add scheduling to detection

### ISSUE THREAD

**Issue by smartelnovo** [jakowenko/double-take#115]
I would be nice, in a busy environment, to add a schedule to disable the detection, I was thinking of something like this to create the schedule:

![image](https://user-images.githubusercontent.com/53451003/135540540-83bca2a1-72cf-41c3-afe0-8f3bed840f01.png)

And then assign it to the camera in the config,  this would avoid filling up the matches during periods that you don't really care, altough we are still recording in Frigate.

**Comment by jakowenko** [comment]
Thanks for the suggestion. I think this makes sense to build in, but may be a little tricky from the UI right now. I could do range sliders like you have above, but it would be nice to support multiple time blocks in a given day and assign cameras to those, etc.

What do you think if I implement this on the `config.yml` side to start, then if that's working I can later look at how to implement it on the UI.

I was thinking something like this for the format.

This allows "disabled" blocks to be created that accepts an array of dates, times, and cameras. You can make as many combinations of these and when a new event triggers Double Take, the API will check to see if it should process the request based on the schedule.

```yaml
schedule:
  disabled:
    - days:
        - monday
        - friday
      times:
        - 12:00-13:00
      cameras:
        - office
        
    - days:
        - tuesday
        - wednesday
      times:
        - 13:00-15:00
      cameras:
        - living-room
```

**Comment by smartelnovo** [comment]
That would be perfect, good idea let's start in the config.yml and see how it goes, it is not something that will change much in time, once the schedules are defined you should not have to change them much, so the gui is not a must for now.

**Comment by jakowenko** [comment]
Hey @smartelnovo, I pushed a new beta build if you'd like to try it out!

Here's the config I was testing with.

The first block disables recognition for the "manual" camera. You should be able to verify it's working by hitting this URL - `localhost:3000/api/recognize?url=https://jakowenko.com/img/david.92f395c6.jpg`

The output should look like this.
```json
{
  "error": "recognition disabled due to schedule",
  "checks": [
    {
      "days": ["monday", "tuesday", "wednesday"],
      "times": ["00:00-23:59"],
      "cameras": ["manual"]
    }
  ]
}
```

```yaml
time:
  # change this to match your timezone unless you want to be in UTC
  timezone: America/Detroit

schedule:
  disable:
    - days:
        - monday
        - tuesday
        - wednesday
      times:
        - 00:00-23:59
      cameras:
        - manual
    - days:
        - tuesday
        - wednesday
      times:
        - 13:00-15:00
      cameras:
        - living-room
 ```

**Comment by smartelnovo** [comment]
I will certainly try it out, and I'll report back.

Thank you!

**Comment by jakowenko** [comment]
Closing since this was released in `1.3.0`
